modules/ui/fields/localized.js

```diff
@@ -470,6 +470,9 @@ export function uiFieldLocalized(field, context) {
             .attr('placeholder', function(d) {
                 return Array.isArray(d.value) ? t('inspector.multiple_values') : t('translate.localized_translation_name');
             })
+            .attr('lang', function (d) {
```

Comment:
The Wikipedia and Wikidata fields would also benefit from this treatment. The Wikipedia field’s language is determined by the language combo box, while the Wikidata field’s language should be available as part of the response from the Wikidata API.

Reply:
:+1: adding Wikipedia and Wikidata fields, trying to keep it minimal changes, but it ought to work